Sources/MapboxMaps/Foundation/MapView.swift

```diff
@@ -326,6 +331,64 @@ open class MapView: UIView {
             mapboxMap: mapboxMap)
     }
 
+    private func subscribeToLifecycleNotifications() {
+        if #available(iOS 13.0, *), bundle.infoDictionary?["UIApplicationSceneManifest"] != nil {
+            notificationCenter.addObserver(self,
+                                           selector: #selector(sceneWillEnterForeground(_:)),
+                                           name: UIScene.willEnterForegroundNotification,
+                                           object: window?.parentScene)
+            notificationCenter.addObserver(self,
+                                           selector: #selector(sceneDidEnterBackground(_:)),
+                                           name: UIScene.didEnterBackgroundNotification,
+                                           object: window?.parentScene)
+        } else {
+            notificationCenter.addObserver(self,
+                                           selector: #selector(appWillEnterForeground),
+                                           name: UIApplication.willEnterForegroundNotification,
+                                           object: nil)
```

Comment:
Per mapbox/mapbox-gl-native#1869, this may be too early for mbgl to wake up and start rendering again:

> UIApplicationWillEnterForegroundNotification is posted before UIApplicationDidBecomeActiveNotification, while the application is still in the background.

/ref mapbox/mapbox-gl-native#1821

Reply:
#1402 replaces this with `UIScene.didActivateNotification`.